game.py

```diff
@@ -0,0 +1,90 @@
+from etc.utils import Color, read_words, get_random_word, is_valid_word
+from collections import defaultdict
+from etc.consts import MAX_ATTEMPTS, INDEX_NOT_FOUND
+
+
+class WordleGame:
+    words = read_words()
+
+    def __init__(self):
+        self.__goal_word = ""
+        self.attempts = 0
+        self.word_found = False
+        self.reset()
+
+    def apply_answer(self, guess):
+        result = None
+
+        if not is_valid_word(guess):
+            raise ValueError("Invalid word", guess)
+        elif self.attempts is MAX_ATTEMPTS:  # Lose
```

Comment:
This works:
![image](https://user-images.githubusercontent.com/67239992/151377013-cd84496a-3c99-4a0c-9dfa-7b4721601947.png)
This doesn't:
![image](https://user-images.githubusercontent.com/67239992/151376941-aa2e2f9b-7c6d-4a3e-9c8b-2df988e482cb.png)

Explain why, then fix your code to compare numbers with `==` 😄

Reply:
Is checks for identity equality (id) and == checks for pointed value equality. For some values, it might work but for the most, it won't.
[Link to explanation source](https://adamj.eu/tech/2020/01/21/why-does-python-3-8-syntaxwarning-for-is-literal/)